.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: node_js
 node_js:
   - "0.10"
-  - "0.8"
```

Comment:
I agree about removing 0.6 but 0.8 is still pretty popular, what was the reason for removing it?